### PR TITLE
Improve code coverage for FileLogTraceListener from simplifying PR #11863 

### PR DESCRIPTION
**Pull request by paul1956**
Improves VB Code Coverage for FileLogTraceListener

## Proposed changes
- Add additional code coverage for FileLogTraceListener

## Customer Impact
- Improve test coverage

## Regression? 
- No

## Risk
- None adds additional tests

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12264)

### CODE REVIEW

File: src/Microsoft.VisualBasic/tests/UnitTests/Microsoft/VisualBasic/Logging/FileLogTraceListenerTests.cs [hunk 1]
```diff
@@ -24,14 +24,17 @@ public void Properties()
         _ = listener.CustomLocation;
     }
 
-    [Fact]
-    public void Write()
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
```
Comment (lonitra) [inline src/Microsoft.VisualBasic/tests/UnitTests/Microsoft/VisualBasic/Logging/FileLogTraceListenerTests.cs:29]: We should have `BoolData` available
```suggestion
    [BoolData]
```

**Comment by paul1956** [comment]
@lonitra can you point to any documentation on BoolData specifically is there a way to specify true, false and null/Nothing for a different test? Web search gets me nothing.

**Comment by lonitra** [comment]
> can you point to any documentation on BoolData specifically is there a way to specify true, false and null/Nothing for a different test? Web search gets me nothing.

`BoolData` attribute is ours and it is defined https://github.com/dotnet/winforms/blob/main/src/Common/tests/TestUtilities/XUnit/BoolDataAttribute.cs. It already creates true/false for you. If you want to also add null/Nothing I believe you can just add `[InlineData(null)]`

**Review by lonitra — APPROVED** [review]
Thank you!

**Pull request merged**